frontend/public/components/dashboard/dashboards-page/cluster-dashboard/health-item.tsx

```diff
@@ -221,10 +222,18 @@ export const PrometheusHealthItem = withDashboardResources<PrometheusHealthItemP
   },
 );
 
-export const ResourceHealthItem: React.FC<ResourceHealthItemProps> = ({ subsystem }) => {
+export const ResourceHealthItem: React.FC<ResourceHealthItemProps> = ({ subsystem, namespace }) => {
   const { title, resources, healthHandler, popupComponent, popupTitle } = subsystem;
-  const resourcesResult = useK8sWatchResources(resources);
-
+  const resourcesResultUnfiltered = useK8sWatchResources(resources);
+  const resourceResultFilteredByNamespace = _.set(
+    _.cloneDeep(resourcesResultUnfiltered),
+    'imageManifestVuln.data',
+    _.filter(
+      resourcesResultUnfiltered.imageManifestVuln.data,
+      (imgvuln: any) => imgvuln?.metadata?.namespace === namespace,
+    ),
+  );
```

Comment:
do we need this?

can we update the resources itself with namespace if have one so that we'll get appropriate data. WDYT?